src/amo/components/Addon/index.js

```diff
@@ -107,8 +115,11 @@ export class AddonBase extends React.Component {
     // of an error.
     if (!errorHandler.hasError()) {
       if (addon) {
-        // eslint-disable-next-line no-restricted-globals
-        if (!isNaN(params.slug)) {
+        const { slug } = params;
+
+        // We want to make sure the slug is invalid before
+        // before we try redirecting.
```

Comment:
`before` appears twice in this sentence.

Reply:
opps..yepp